.travis.yml

```diff
@@ -25,6 +25,7 @@ env:
     - TASK="CI=true TRAVIS=true rake test:integration"
     - TASK="CI=true TRAVIS=true rake test:functional"
     - TASK="CI=true TRAVIS=true rake assets:precompile"
+    - TASK="CI=true TRAVIS=true bundle exec rubocop"
```

Comment:
I still dislike the idea of Rubocop blocking travis builds :/

Reply:
We need to have those since I have got rubocop synch'd with CodeClimate, we're getting a wider view of our PR statuses.